### DISCUSSION TITLE
Losing fetchd setup every time I come back to develop

### DISCUSSION THREAD

**Discussion by bgoober** [fetchai/fetchd#243] [Developers] [answered]
Hello all, 

Every time I come back to develop my agent I seem to lose the ability to interact with fetchd, although the repo remains cloned on my device. 

Specifically, I have to navigate back into the fetchd folder and run the following commands in order to successfully query which version of fetchd I am running, and its relative path (These commands are found in [Building the Ledger](https://docs.fetch.ai/ledger_v2/building/#step-1-setup)) :

`make build`
`make install`
`export PATH=$PATH:~/go/bin`
`source ~/.bashrc`

At this point I can successfully run `which fetchd` and `fetchd version` and see the correct outputs. However, it seems that I have to do this EVERY time that I turn off/on my computer and come back to develop my agent. A friend is having this exact same experience/issue on his machine as well. We are both running Ubuntu 20+ LTS. 

How can I avoid having to re-setup fetchd every time I come back to do some dev work? Thank you!

**Comment by daeMOn63** [comment, marked as answer]
Hello, just add the `export PATH=$PATH:~/go/bin` line at the end of the `~/.bashrc` file

**Reply by bgoober** [reply]
Thank you, I think this solved it.
